myuw/data/resource_link_import.csv

```diff
@@ -1,154 +1,168 @@
+Teaching,Course Materials,all,https://www.ubookstore.com/faculty,Order Textbooks,,,,,,,no
+Teaching,Course Materials,all,https://finance.uw.edu/c2/printing-copying/course-packs,Request Course Packs,,,,,,,no
+Teaching,Course Materials,all,https://lib.washington.edu/types/course/instructors/request,Request Course Reserves,,,,,,,no
+Teaching,Grading,all,https://gradepage.uw.edu,GradePage,,,,,,,no
+Teaching,Grading,all,https://apps.registrar.washington.edu/grade-change/pages/change.php,Change Submitted Grades,,,,,,,no
+Teaching,Grading,all,,,http://depts.washington.edu/grading,Faculty Resource on Grading,https://www.uwb.edu/registration/policies/grading,Grading Policies,https://www.tacoma.uw.edu/uwt/registrar/grading-policy,Grading Policy,no
+Teaching,Tools,all,https://canvas.uw.edu,Canvas LMS,,,,,,,no
+Teaching,Tools,all,https://catalyst.uw.edu/,Catalyst Web Tools,,,,,,,no
+Teaching,Tools,all,https://uw.hosted.panopto.com,Panopto Lecture Capture,,,,,,,no
+Teaching,Tools,all,https://itconnect.uw.edu/learn/tools/polleverywhere/,Poll Everywhere,,,,,,,no
+Teaching,Tools,all,https://coda.uw.edu,Course Stats,,,,,,,no
+Teaching,Tools,all,https://washington.zoom.us,Zoom,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/,UW Research Website,,,,,,,no
+Research,Research Administration,all,,,https://www.washington.edu/research/new-to-uw-research/,New to UW Research,https://www.uwb.edu/research/support,New to UW Research,https://www.tacoma.uw.edu/or/conducting-research-uw-tacoma,New to UW Research,no
+Research,Research Administration,all,https://www.washington.edu/research/myresearch-lifecycle/,MyResearch Project Lifecycle,,,,,,,no
+Research,Research Administration,all,https://myresearch.washington.edu/,MyResearch Portal,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/sage/,System to Administer Grants Electronically (SAGE),,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/,Research Administration Tools,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/resources/funding-opportunities/,Funding Opportunities,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/research-centers/,Research Centers and Institutes,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/shared-research-facilities-resources/,Shared Research Facilities and Resources,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://escience.washington.edu/,eScience Institute,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/services/faculty,UW Libraries: Getting started with your research,,,,,,,no

```

Comment:
This title/page combo is a bit confusing to my eyes (may not be for a researcher). It appears to go to a generic UW Libraries page on how to conduct library research. Is that the intention? If so, we may want to specify it's about getting started with *library* research...

Reply:
Updated to "Getting started with library research" !